VAMobile/src/components/Menu/MenuView.tsx

```diff
@@ -29,11 +31,9 @@ export type MenuItemActionsType = {
   /** adds a divider after the action*/
   addDivider: boolean
   /** name of the icon to show */
```

Comment:
Comment isn't updated to match the change

Reply:
updated